test/t/test_fprintd_enroll.py

```diff
@@ -13,6 +15,9 @@ def test_basic(self, completion):
     def test_options(self, completion):
         assert completion
 
+    @pytest.mark.xfail(
```

Comment:
```suggestion
    @pytest.mark.skipif(
```

I think we should be skipping and not xfailing tests in scenarios like this. We seem to have the same problem in at least some of our xfail vs skip uses and should review and fix them all up in another PR. (For example, in strict xfail mode, an unexpected pass would cause the tests to fail.)

- https://docs.pytest.org/en/stable/how-to/skipping.html
- https://docs.pytest.org/en/stable/how-to/skipping.html#strict-parameter

Pre-approved with that addressed.